app/controllers/tickets_controller.rb

```diff
@@ -0,0 +1,66 @@
+class TicketsController < ApplicationController
+  protect_from_forgery with: :null_session
+  after_action :set_points, only: :create
+  def create
+    @ticket = Ticket.create(selection_nested_params(ticket_params))
+    if @ticket.save!
+      redirect_to @ticket
+    end
```

Comment:
Avoid of using `if` without `else` statement. Avoid of using `if` with only one line inside.